test/images/agnhost/serve-hostname/serve_hostname.go

```diff
@@ -15,7 +15,8 @@ limitations under the License.
 */
 
 // A small utility to just serve the hostname on TCP and/or UDP.
-package main
+
+package servehostname
 
 import (
 	"flag"
```

Comment:
Is this import still necessary with the use of Cobra flags? Likewise for the other binaries in this PR.

Reply:
Removed.